src/main/java/com/cms/claimmanagement/insuranceCompany/policy/Controller/PolicyController.java

```diff
@@ -8,18 +8,18 @@
 import java.util.List;
 
 @RestController
-@RequestMapping("/v1/policy")
+@RequestMapping("/v1/policies")
```

Comment:
Great mappings! Using the HTTP methods to  access the resource. :)

Reply:
nice!